### DISCUSSION TITLE
Source Date

### DISCUSSION THREAD

**Discussion by StewartM182** [codefori/vscode-ibmi#2187] [Q&A]
👉🏻 Can you add a feature to search for source dates like PDM. Enter a Search Date and search for Less than, Equal to , or Greater than.
The Source Date should not be included, or altered as part of the source code either.
There should be a way to initialize the source line dates to i.e. '000000' like PDM would be helpful.
There should be a way to display and use the source code lines instead of the Visual Code lines.

<img width="773" alt="Screenshot 2024-07-19 115743" src="https://github.com/user-attachments/assets/4c1f916b-1c7b-4ed2-b589-f87e16307442">

**Comment by worksofliam** [comment]
Converted to a discussion instead of an issue.

Did you see our docs on source dates? https://codefori.github.io/docs/developing/sourcedates
